src/parser/demonhunter/havoc/modules/talents/BlindFury.js

```diff
@@ -51,7 +51,7 @@ class BlindFury extends Analyzer{
   suggestions(when) {
     when(this.suggestionThresholds)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest(<>Try to cast <SpellLink id={SPELLS.EYE_BEAM.id} /> at 30 to 50 fury. Having more than 50 fury at cast is considered a dps loss.</>)
+        return suggest(<>Try to cast <SpellLink id={SPELLS.EYE_BEAM.id} /> at 30 to 50 fury. Having more than 50 fury at cast leads to to much fury wasted and a dps loss.</>)
```

Comment:
```suggestion
        return suggest(<>Try to cast <SpellLink id={SPELLS.EYE_BEAM.id} /> at 30 to 50 fury. Having more than 50 fury at cast leads to too much fury wasted and a dps loss.</>)
```

Minor typo, but even with that fixed I think this wording could use some work. How about `Having more than 50 Fury at cast leads to wasting Fury which is a DPS loss.`? This also has Fury capitalized, as I try to capitalize spell/resource names so it's more obvious they refer to the in-game spells, and DPS since it's an abbreviation that's most commonly capitalized.

Reply:
updated to the following:
Cast <SpellLink id={SPELLS.EYE_BEAM.id} /> with 50 or less Fury to minimize Fury waste and maximize DPS.